### PR TITLE
Add a new keyboard: NQG

### PR DESCRIPTION
**Pull request by culturalsnow**
## Description

Adding a new keyboard to QMK.
NQG is a 30% ortholinear keyboard with an additional macro key and a row for thumb keys.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
You can remove the `.gitignore` file.

**Comment by culturalsnow** [comment]
> You can remove the `.gitignore` file.

Removed. Thanks!

File: keyboards/emptystring/NQG/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# NQG: Not Quite Gherkin
+
+![NQG](https://github.com/culturalsnow/NQG/blob/master/images/NQGLayout.jpg)
+
+NQG (Not Quite Gherkin) is a 30% ortholinear keyboard with a macro key and dedicated row for thumb keys, made by emptystring studio.
+
+Keyboard Maintainer: [SA_EndlessGame](http://twitter.com/SA_EndlessGame) on twitter
```
Comment (mechmerlin) [inline keyboards/emptystring/NQG/readme.md]: We typically have this set as a link to the maintainer's github account.

Comment (culturalsnow) [reply keyboards/emptystring/NQG/readme.md]: Modified link to point to my github account instead.
Thank you!

**Pull request merged**